web/settings.py

```diff
@@ -45,14 +47,24 @@ def get_secret(secret_name, region_name):
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.getenv("DEBUG", "False") == "True"
 
+# AWS fix to add the IP of the AWS Instance to ALLOWED_HOSTS
 hosts_list = os.getenv("DOMAIN", "localhost:brain-score-web-dev.us-east-2.elasticbeanstalk.com").split(":")
-hosts_list.append("Brain-score-web-prod-updated.kmk2mcntkw.us-east-2.elasticbeanstalk.com")  # updated prod env
+hosts_list.append("www.brain-score.org")
+hostname = socket.gethostname()
+local_ip = socket.gethostbyname(hostname)
+hosts_list.append(local_ip)
+try:
+    reqToken = Request('http://169.254.169.254/latest/api/token', method='PUT')
+    reqToken.add_header('X-aws-ec2-metadata-token-ttl-seconds', '21600')
+    token = urlopen(reqToken).read().decode("utf-8")
+    reqIP = Request('http://169.254.169.254/latest/meta-data/local-ipv4')
+    reqIP.add_header('X-aws-ec2-metadata-token', token)
+    ipv4 = urlopen(reqIP).read().decode("utf-8")
+    hosts_list.append(ipv4)
+except:
+    hosts_list = hosts_list
```

Comment:
is this from another PR? or is this necessary for django/python version fix?

Reply:
This was from an old PR t (#199) that still needed to be merged, so I figured it would go along with this change. It fixed some of the AWS env stuff

Reply:
approved #199, merge that first, then this PR here should be only few changes right?